src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -564,23 +567,31 @@ namespace AppInstaller::Runtime
         case PathName::Temp:
         case PathName::DefaultLogLocation:
         {
-            result.Path = GetPathToUserTemp();
+            if (forDisplay)
+            {
+                result.Path.assign("%TEMP%");
+            }
+            else
+            {
+                result.Path = GetPathToUserTemp();
+            }
+
             result.Path /= s_DefaultTempDirectory;
             result.Path /= GetRuntimePathStateName();
+
             if (path == PathName::Temp)
             {
                 result.SetOwner(ACEPrincipal::CurrentUser);
                 result.ACL[ACEPrincipal::System] = ACEPermissions::All;
                 result.ACL[ACEPrincipal::Admins] = ACEPermissions::All;
             }
+
+            if (path == PathName::DefaultLogLocation && forDisplay)

```

Comment:
```suggestion
            if (forDisplay)

```

I understand you were trying to keep previous behavior, but for this one, forDisplay itself seems enough